fury/ui/core.py

```diff
@@ -730,23 +732,33 @@ def __init__(
             Adds text shadow.
         size : (int, int)
             Size (width, height) in pixels of the text bounding box.
+        auto_font_scale : bool, optional
+            Automatically scale font according to the text bounding box.
+        dynamic_bbox : bool, optional
+            Automatically reisize the bounding box according to the content.
```

Comment:
Shouldn't these new parameters also be specified as attributes of `TextBlock2D`?

Reply:
Maybe yes!
Will update it accordingly.